Backpack/src/main/res/values/backpack.text.xml

```diff
@@ -45,6 +45,10 @@
     <item name="android:letterSpacing">-0.0125</item>
   </style>
 
+  <style name="bpkTextBaseRelative" parent="bpkTextBase">
```

Comment:
I am 90% sure that this actually doesn't do anything as the line spacing multiplier isn't part of the text appearance, but to avoid breaking some usages outside of our components I kept these